examples/fabric/cloud_router/go/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 toolchain go1.22.6
 
 require (
-	github.com/equinix/pulumi-equinix/sdk 0.15.1
+	github.com/equinix/pulumi-equinix/sdk 0.16.1
```

Comment:
These PRs might indicate further issues with our automation.  I'm not sure exactly when these examples should have been updated, but it certainly seems like these changes should have been committed before release and not after.

@ocobles Any ideas where we might be able to generate and commit these changes so that they go out as part of the release instead of after?

Reply:
It looks like running `make generate_examples` produces these changes, but I'm not sure if there's anywhere to hook into the release process to ensure that command is executed.  It's possible these updates can't happen until after the release?

This `upgrade-provider` issue seems related but is light on details: https://github.com/pulumi/upgrade-provider/issues/246.  We may just have to accept this gap in example generation for now.